src/data/Api.js

```diff
@@ -0,0 +1,19 @@
+const list = [];
+
+const hello = async () => {
+  let url = "http://localhost:3001/list/all";
```

Comment:
create .env file and pick this backend url from environment variables.

https://create-react-app.dev/docs/adding-custom-environment-variables/